cmd/notation/cert/cleanupTest.go

```diff
@@ -0,0 +1,116 @@
+package cert
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/notaryproject/notation-go/config"
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation/cmd/notation/internal/truststore"
+	"github.com/notaryproject/notation/internal/osutil"
+	"github.com/spf13/cobra"
+)
+
+type certCleanupTestOpts struct {
+	keyName string
+}
+
+func certCleanupTestCommand(opts *certCleanupTestOpts) *cobra.Command {
+	if opts == nil {
+		opts = &certCleanupTestOpts{}
+	}
+	command := &cobra.Command{
+		Use:   "cleanup-test <common_name>",
```

Comment:
It seems the `notation cert cleanup-test` command is not behind an experimental flag yet.

Reply:
> It seems the `notation cert cleanup-test` command is not behind an experimental flag yet.

Yes, this is something that we need to discuss. Since we decided to not put `notation cert generate-test` behind `experimental`, I think we shouldn't put `notation cert cleanup-test` behind the flag either? This is to make the user experience more consistent before and after.

Reply:
@Two-Hearts @yizha1 @FeynmanZhou Let's discuss it in the community meeting.